app/models/work_package/pdf_export/standard.yml

```diff
@@ -0,0 +1,176 @@
+  margin_top: 60
+  margin_bottom: 60
+  margin_left: 36
+  margin_right: 36
+  page_break_threshold: 200
+  link_color: "175A8E"
+
+page_header:
+  offset: 20
+  size: 8
+
+page_footer:
+  offset: -30
+  size: 8
+
+page_logo:
+  height: 20
+  align: "right"
+
+page_heading:
+  size: 14
+  styles: [ "bold" ]
+  margin_bottom: 10
+
+overview:
+  group_heading:
+    size: 11
+    styles: [ "bold" ]
+    margin_bottom: 10
+  table:
+    subject_indent: 8
+    margin_bottom: 20
+    cell:
+      size: 9
+      color: "000000"
+      padding_left: 5
+      padding_right: 5
+      padding_top: 0
+      padding_bottom: 5
+      border_width: 0.25
+    cell_header:
+      size: 9
+      styles: [ "bold" ]
+    cell_sums:
+      size: 8
+      styles: [ "bold" ]
+
+toc:
+  subject_indent: 4
+  margin_top: 10
+  margin_bottom: 20
+  item:
+    size: 9
+    color: "000000"
+    margin_bottom: 4
+  item_level_1:
+    size: 10
+    styles: [ "bold" ]
+    margin_top: 4
+    margin_bottom: 4
+  item_level_2:
+    size: 10
+
+work_package:
+  margin_bottom: 20
+  subject:
+    size: 14
+    styles: [ "bold" ]
+    margin_bottom: 10
+  label:
+    size: 11
+    styles: [ "bold" ]
+  attributes_table:
+    margin_bottom: 10
+    cell:
+      size: 9
+      color: "000000"
+      padding_left: 5
+      padding_right: 5
+      padding_top: 0
+      padding_bottom: 5
+      border_color: "4B4B4B"
+      border_width: 0.25
+    cell_label:
+      styles: [ "bold" ]
+  section:
+    heading:
+      size: 9
+      styles: [ "bold" ]
+      margin_top: 4
+      margin_bottom: 4
+    item:
+      size: 9
+      styles: [ "italic" ]
+      margin_left: 10
+  markdown_label:
+    size: 12
+  markdown_margin:
+    margin_top: 12
+    margin_bottom: 8
+  markdown:
+    font:
+      size: 10
+      leading: 3
+    header:
+      size: 8
+      styles: [ "bold" ]
+    header_1:
+      size: 10
+      padding_bottom: 4
+    header_2:
+      size: 10
+    header_3:
+      size: 9
+    paragraph:
+      align: "left"
+    unordered_list:
+      spacing: 1
+    unordered_list_point:
+      spacing: 4
+    ordered_list:
+      spacing: 1
+    ordered_list_point:
+      spacing: 4
+    task_list:
+      spacing: 1
+    task_list_point:
+      spacing: 4
+      checked: "☑"
+      unchecked: "☐"
+    link:
+      color: "175A8E"
+      styles: [ ]
+    code:
+      color: "880000"
+      size: 9
+      font: "SpaceMono"
+    blockquote:
+      background_color: "f4f9ff"
+      size: 10
+      styles: [ "italic" ]
+      color: "0f3b66"
+      border_color: "b8d6f4"
+      border_width: 1
+      padding: 4
+      padding_left: 6
+      margin_top: 4
+      margin_bottom: 4
+      no_border_left: false
+      no_border_right: true
+      no_border_bottom: true
+      no_border_top: true
+    image:
+      align: "left"
+    codeblock:
+      background_color: "F5F5F5"
+      color: "880000"
+      padding: "3mm"
+      size: 8
+      margin_top: "2mm"
+      margin_bottom: "2mm"
```

Comment:
❓ Is there a reason we are adding units here and no tfor the other margins?

Reply:
No good reason other than it's possible to use units. ^^
I unified the usage to have non specified. (PostscriptPoint is the default)